### PR TITLE
feat(dedicated.network-securit): security dashboard feedback

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-503
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

- rework on services and IPs selection with list or search components
- scrubbing center log - no action available if event older than 2 weeks
- scrubbing center log - default period: 1 year
- traffic chart - use "none" label instead of 0 value into resume panel
- traffic chart - display date and use 24h format
- traffic chart - default period: 2 weeks

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js [hunk 1]
```diff
@@ -98,35 +103,41 @@ export default class ScrubbingCenterController {
           this.autocomplete,
         )
         .then((data) => {
-          this.autocomplete = data;
-          this.getAllEvents('selectService');
+          this.ipsList = data.map((el) => el.ipBlock);
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:106]: **nitpick:** Avoid short names
```suggestion
          this.ipsList = data.map(({ ipBlock }) => ipBlock);
```

File: packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js [hunk 2]
```diff
@@ -58,36 +61,31 @@ export default class TrafficController {
           this.autocomplete,
         )
         .then((data) => {
-          this.autocomplete = data;
+          this.ipsList = data.map((el) => el.ipBlock);
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js:64]:
```suggestion
          this.ipsList = data.map(({ ipBlock }) => ipBlock);
```

File: packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js [hunk 3]
```diff
@@ -98,35 +103,41 @@ export default class ScrubbingCenterController {
           this.autocomplete,
         )
         .then((data) => {
-          this.autocomplete = data;
-          this.getAllEvents('selectService');
+          this.ipsList = data.map(({ ipBlock }) => ipBlock);
+          this.selectedIp = this.ipsList;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:107]: **question:** Are you trying to set the `selectedIp` with entire `ipList`?

Comment (Steffy29) [reply packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js:107]: yes, I am

**Review by BrunoMarquesOVH — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**